chemprop/featurizers/molecule.py

```
@@ -93,3 +93,22 @@ def __init__(self):
 class V1RDKit2DNormalizedFeaturizer(V1RDKit2DFeaturizerMixin):
     def __init__(self):
         self.generator = rdNormalizedDescriptors.RDKit2DNormalized()
+
+
+@MoleculeFeaturizerRegistry("charge")
+class ChargeFeaurizer(VectorFeaturizer[Mol]):
+    def __call__(self, mol: Chem.Mol) -> np.ndarray:
+        return np.array([Chem.GetFormalCharge(mol)])
+
+    def __len__(self) -> int:
+        return 1
+
+
+@MoleculeFeaturizerRegistry("multiplicity")
+class MultiplicityFeaurizer(VectorFeaturizer[Mol]):
+    def __call__(self, mol: Chem.Mol) -> np.ndarray:
+        mult = sum(atom.GetNumRadicalElectrons() for atom in mol.GetAtoms()) + 1
+        return np.array([mult])
```

Will this work on excited state molecule, especially transition metal complexes? If not, it should raise an error.

---

seems to also be a typo here: `Featurizer` rather than `Feaurizer` for both of these